osc-server/src/main/java/org/osc/core/broker/service/mc/AddApplianceManagerConnectorService.java

```diff
@@ -158,7 +163,18 @@ void checkManagerConnection(DryRunRequest<ApplianceManagerConnectorRequest> requ
             try {
                 this.apiFactoryService.checkConnection(mc);
             } catch (Exception e) {
-                ErrorTypeException errorTypeException = new ErrorTypeException(e, ErrorType.MANAGER_CONNECTOR_EXCEPTION);
+
+                Throwable rootCause = ExceptionUtils.getRootCause(e);
+                Throwable cause = e;
+
+                if (rootCause instanceof SocketException
+                        || rootCause instanceof SSLException
+                        || rootCause instanceof ForbiddenException
```

Comment:
looks like we cannot handle unauthorized case without adding checked exceptions to the API package. So just handle the connection exceptions case